### ISSUE TITLE
I believe "All Rights Reserved" is obsolete as of 20+ years now, and is in conflict with LGPL anyway.

### ISSUE THREAD

**Issue by CharlesScoville** [me-no-dev/AsyncTCP#149]
Source files are headed with copyright notices that claim both LGPL, and "All Rights Reserved." 

"All Rights Reserved" is an antique legal term that [hasn't had legal meaning since 2000](https://en.wikipedia.org/wiki/All_rights_reserved#Obsolescence). Furthermore, if the phrase did have legal meaning, it would seem to be in direct conflict with the LGPL license, since the LGPL specifically grants some rights, where reserving all rights would forbid them.

I expect this was a simple oversight born from following old, well intended, conventions. My fear, however, is the person placing this copyright notice could be (or eventually become!) malicious and repossess ownership of the project using this as a loophole. I concede that it's also possible there is fear of contributor takeover (i.e. people submit source under their own license, then make demands) 

In all cases, I would strongly urge contributors agree to remove this wording, as it's at the very least confusing, if not outright contradictory. If hostile takeover is a major concern, I recommend having a separate ["Contributor License Agreement"](https://en.wikipedia.org/wiki/Contributor_License_Agreement), rather than trying to get tricky with the copyright notice.

Bottom line: I, for one, would be very reluctant to contribute to a project with copyright notices left as is here. my2c
